### ISSUE TITLE
CSV DateTime uses InvariantCulture resulting in swapped day/months in CSVDestination

### ISSUE THREAD

**Issue by bencassie** [roadrunnerlenny/etlbox#34]
Hi,

I have found that CSVWriter is configured using Invariant Culture leading to an issue when serializing DateTimes. 

For example, I have a date 2nd May 2019, yet when written with CSVDestination it is serialized to 5th Feb 2019. 

See here -
https://github.com/JoshClose/CsvHelper/issues/697 and here https://stackoverflow.com/questions/46778141/datetime-formats-used-in-invariantculture

**Comment by bencassie** [comment]
Have tested adding the line mentioned in the CSVHelper (below) in CSVDestination.InitObjects ok.

csv.Configuration.TypeConverterOptionsCache.GetOptions<DateTime>().Formats = new []{"yyyy-MM-dd HH:mm:ss"};

**Comment by roadrunnerlenny** [comment]
Hi Ben,

thanks again for finding this issue. 
Actually, I just tested this behavior, and when I create a DateTime object for Feb 5th 2010 (`new DateTime(2010, 02, 05)`) , the CSVWriter would produce "02/05/2010 00:00:00".
Which would actually be the correct date within the Unites States, but for me as somebody from Germany this also doesn't look right. 
So, first of you all you don't need to change the InitObjects() method in CSVDestination to override this behaviour. The Configuration property is public and exposed within the CSVDestination, so the following code would also work:
```
CSVDestination<MySeriRow> dest = new CSVDestination<MySeriRow>("./DateTimeSerialization.csv");
dest.Configuration.TypeConverterOptionsCache.GetOptions<DateTime>().Formats = new[] { "yyyy-MM-dd HH:mm:ss" };
```
This should help you in the first place. 

But you are right, this is actually not the desired behaviour - acutally it would be better to have a real region and country independent date in the output. I will add this for the next version, so the InitObjects will actually output the format you suggested (though I will also include the milliseconds).

**Comment by bencassie** [comment]
Hi,

Yes accessing their configuration via the destination is fine thank you!

It definitely seems like an unexpected behavior for people that aren't keeping an eye on the output though!

Thanks

**Comment by roadrunnerlenny** [comment]
Just a quick update: I just released version 1.6.2 which sets the default date format when exporting to `"yyyy-MM-dd HH:mm:ss.fff"`.
See https://www.nuget.org/packages/ETLBox/1.6.2.
